SMLHelper/Patchers/ItemsContainerPatcher.cs

```diff
@@ -0,0 +1,111 @@
+﻿namespace SMLHelper.V2.Patchers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Reflection;
+    using Harmony;
+
+    // This entire patcher is only here for performance reasons.
+    // This is not intended to be exposed by the public API.
+    internal static class ItemsContainerPatcher
+    {
+        private class StorageCache : Dictionary<Vector2int, bool>
```

Comment:
Why make an empty class just to represent this? Wouldn't it be better to just have that stated explicitly in the main declaration of the cache?

Reply:
Mostly for readability.  
But it is entirely optional and likely doesn't have any real hit to performance.

Reply:
There. No more empty class.